docs/get-started/developers/installation.md

```diff
@@ -24,36 +24,60 @@ information with code snippets that are specific to that element.
 
 ### Red Hat CDN
 
-<rh-alert state="warning">
-  <h4 slot="header">CDN Prerelease</h4>
-  <p>We are currently working on our CDN, which will be soon moving into beta. 
-    This will be the preferred method of installation in the near future. If you 
-    are a Red Hat associate and have questions or comments about the CDN or 
-    installation process please connect with us on Slack.</p>
+<rh-alert state="info">
+  <h4 slot="header">CDN v2 for v1 users</h4>
+  <p>If you previously used the v1 CDN, and want to migrate to point to the new server the v1 files are still available under 
+  <a href="https://www.redhatstatic.com/dssf-001/v1/">https://www.redhatstatic.com/dssf-001/v1/</a>. To migrate, change 
+  <code>/dx/v1-alpha/</code> to <code>/dssf-001/v1/</code> in your current URLs.</p>
+  <p>This new server will be the preferred method of installation going forward. If you are a Red Hat associate 
+  and have questions or comments about the CDN or installation process please connect with us on Slack.</p>
 </rh-alert>
 
-The recommended way to load RHDS is via the Red Hat Digital Experience CDN, and 
+The recommended way to load RHDS is via the [Red Hat Digital Experience CDN](https://www.redhatstatic.com/dssf-001), and 
```

Comment:
it's worth mentioning here that this option is only available to red hat projects, on pre-authorized red hat domains